### ISSUE TITLE
About video-level probability

### ISSUE THREAD

**Issue by yangjiangeyjg** [Pilhyeon/Learning-Action-Completeness-from-Points#2]
Thanks for your excellent job! 
I am confused why express video-level probability by that:
_vid_score = (torch.mean(topk_scores, dim=1) * vid_labels) + (torch.mean(cas_sigmoid[:,:,:-1], dim=1) * (1 - vid_labels))_
Inconsistent between training and testing.

**Comment by Pilhyeon** [comment]
Hi, thanks for your interest.
It is sort of a technical trick for suppressing the scores for negative action classes (absent in the video).
Instead of selecting only top k segments, we here uniformly propagate the gradients from negative classes into all segments.
We find that this modification during training slightly increases the performance, though not significant.

**Comment by yangjiangeyjg** [comment]
> Hi, thanks for your interest.
> It is sort of a technical trick for suppressing the scores for negative action classes (absent in the video).
> Instead of selecting only top k segments, we here uniformly propagate the gradients from negative classes into all segments.
> We find that this modification during training slightly increases the performance, though not significant.

Thanks.
